sw/host/opentitantool/h1dx_devboard.json

```diff
@@ -0,0 +1,23 @@
+{
+  "uarts": [
+    {
+      "name": "console",
+      "baudrate": 115200,
+      "parity": "None",
+      "stopbits": "Stop1"
+    }
+  ],
+  "flash": [
+    {
+      "name": "H1Dx_flash",
+      "reset_pin": "RESET",
+      "bootloader_pin": "BOOTSTRAP",
+      "driver": {
+	"SpiExternalDriver": {
+	  "command": "/home/jbk/src/ti50/ports/cr50-utils/software/tools/SPI/spiflash --dauntless --verbose --parent --input=$IMAGEFILE",
```

Comment:
This file should perhaps remain private to you because it references this tool in your home and the commandline switches refer to a chip by codename.

Reply:
Good point, I failed to realize that the name of the file was not the only problem.  I will keep them out of this PR, and reconsider at a later time.